tests/processes/ProcessingTest.py

```diff
@@ -816,3 +817,37 @@ def test_autoapply_override(self):
                                       self.section, self.log_printer, {}, [],
                                       console_printer=self.console_printer)
         self.assertEqual(newres, [])
+
+
+class apply_patchesTest(unittest.TestCase, Bear):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.section.append(Setting('default_actions', '**: ApplyPatchAction'))
+        self.log_printer = LogPrinter(ConsolePrinter(), log_level=0)
+
+    def test_autoapply_actions_DummyBear(self):
+        """
+        Test to ensure --apply-patches works for ESLintBear.
+        """
+        with TempDirectory() as d:
+            d.write('test.js', ''.encode())
+            file = os.path.join(d.path, 'test.js')
+            self.file_dict = {file: ['a']}
+            self.new_output = {file: ['b']}
+            self.result = Result('YBear', 'msg1')
+            self.result.origin = 'DummyBear (dummy/dummy)'
+            lines = ''.join(self.file_dict[file])
+            output = ''.join(self.new_output[file])
+            self.result.diffs = dict([(file,
+                                       Diff.from_string_arrays(
+                                            lines.splitlines(True),
+                                            output.splitlines(True)))])
+            self.result = [self.result]
+            ret = autoapply_actions(self.result,
+                                    self.file_dict,
+                                    {},
+                                    self.section,
+                                    self.log_printer)
+
+        self.assertEqual(ret, [])
```

Comment:
Okay I have a couple of doubts can you help me clarify. The basic aim is to check if no output is printed with a js file having error when you are telling coala to fix the changes automatically. Like where is the test.js file containing the error :sweat_smile:  . I know this might be a very stupid doubt but if you could help me clarify it